### ISSUE TITLE
Inconsistent behaviour of preview/render view modes

### ISSUE THREAD

**Issue by bobc** [openscad/openscad#1419]
Preview (F5) and Render (F6) seem quite clear and straightforward, there are basically two modes.

But then the first 5 options under View are confusing and inconsistent. If I do a Render, View|Preview becomes unchecked as expected. But if I then select View|Preview, the display blanks. I would expect it to run Preview mode again automatically, but I have to do F5.

In Preview mode, Show Edges display edges. If I then select Wireframe, it switches to Render, but displays a wireframe of the last cached CGAL geometry (even if flush caches is used). Except if the Render was a polyset, in which case the solid view (no edges) is shown!

Show Edges has a different look depending on whether it is in Preview or Render Mode.

View|Thrown Together has more quirks. Used in Render mode, it switches to Preview, but the display blanks. Thrown Together is checked and Preview isn't, but Thrown Together is a mode of Preview.

Proposal:
- I think it should be made clear whether the mode is Preview or Render, with an indicator in title bar/panel
- The View options should not change mode. 
- The View options should attempt to do the same thing in any mode. 
- View options that do not apply to the current mode should be greyed and not selectable.
- Wireframe and Show Edges are very similar, it is only necessary to have Wireframe(on/off) and Surfaces(on/off). 
##

**Comment by kintel** [comment]
I agree. The goal is to clean up the OpenGL rendering code properly, and update the GUI to be more consistent then.

**Comment by kintel** [comment]
We've refactored the UI related to this quite a bit recently. I'll close this as we solved some of the reported issues. If there are still outstanding issues, please open a new ticket.
